oslabs-beta/Kubiquity
# Add logo

WHAT DOES THIS PR DO:
1. Add logo to splash page and homepage. 
2. Remove old server. 
3. Move stylesheets into assets directory. 

HOW TO TEST: 
1. (if you're fine not seeing the data): `gh pr checkout 40 && npm install && npm run dev`
2. Verify that the app still is styled.
